string name
trajectory_msgs/JointTrajectoryPoint start
trajectory_msgs/JointTrajectoryPoint goal
float64 tau